src/new-client/src/components/DraggableTearOut/canDrag.ts

```diff
@@ -1 +1,2 @@
-export const canDrag = true
+import { supportsTearOut } from "@/App/TearOutSection/supportsTearOut"
+export const canDrag = true && supportsTearOut
```

Comment:
Minor - Can we add a comment here, it's a bit confusing. Something like
`If tear out is supported, allow drag`.